pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -230,4 +233,7 @@ func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfigura
 	if obj.VolumePluginDir == "" {
 		obj.VolumePluginDir = DefaultVolumePluginDir
 	}
+	if obj.RootDir == "" {
+		obj.RootDir = kubeletconfigv1beta1.DefaultRootDir
```

Comment:
should the `DefaultRootDir` const be in this package, peer to `DefaultVolumePluginDir`?